packages/teleport/src/cluster/components/Audit/Audit.tsx

```diff
@@ -71,13 +70,17 @@ export default function Audit() {
       )}
       {isSuccess && (
         <Switch>
-          <Route title="Sessions" path={sessionsRoute}>
+          <Route title="Sessions" path={cfg.routes.clusterAuditSessions}>
             <AuditSessions pageSize={20} events={events} />
           </Route>
-          <Route title="Events" path={eventsRoute}>
+          <Route title="Events" path={cfg.routes.clusterAuditEvents}>
             <AuditEvents events={events} />
           </Route>
-          <Redirect exact from={auditRoute} to={sessionsRoute} />
+          <Redirect
+            exact
+            from={cfg.routes.clusterAudit}
+            to={cfg.routes.clusterAuditSessions}
+          />
```

Comment:
I wasn't sure of this change. I needed to make this change so that the router can extract the `clusterId` from the URL from the property `params` in `computedMatch` object.

Reply:
discussed over slack